xarray/core/formatting.py

```diff
@@ -428,13 +395,14 @@ def _mapping_repr(
 
 def coords_repr(coords, col_width=None, max_rows=None):
     if col_width is None:
-        col_width = _calculate_col_width(_get_col_items(coords))
+        col_width = _calculate_col_width(coords)
     return _mapping_repr(
         coords,
         title="Coordinates",
-        summarizer=summarize_coord,
+        summarizer=summarize_variable,
         expand_option_name="display_expand_coords",
         col_width=col_width,
+        indexes=coords.indexes,
```

Comment:
This raises an error for custom indexes (which are not convertible to `pandas`). Shouldn't it be this instead:
```suggestion
        indexes=coords.xindexes,
```